src/Security/Group.php

```diff
@@ -115,6 +115,11 @@ public function getAllChildren()
      */
     public function getCMSFields()
     {
+        $list = Group::get()->exclude('ID', $this->ID);
+        $groups = new ArrayList();
+        foreach ($list as $grp) {
+            $groups->push(['ID' => $grp->ID, 'Title' => html_entity_decode($grp->Breadcrumbs)]);
```

Comment:
Are you able to use `Convert::xml2raw($group->Breadcrumbs)` here? If so, that'd centralise the use of this function a bit more, particularly since it specifies encoding options